src/lib/components/phoneInput/phoneInput.js

```diff
@@ -21,12 +23,12 @@ class PhoneInput extends PureComponent {
 
   static defaultProps = {
     ...defaultComponentPropTypes,
-    placeholder: 'phone',
+    placeholder: "phone",
     className: styles.input,
-    name: 'phone',
+    name: "phone",
     required: false,
-    defaultCountry: 'auto',
-    value: '',
+    defaultCountry: "auto",
+    value: "",
```

Comment:
we should use single quotes, right?

Reply:
I agree...I think I updated my vscode settings and it is not checking lint errors anymore...I would like to check props and onChange method mainly just to close this component and merge to master, because we need it also on the website @davidpaley

Reply:
Sorry, I have to spend some time fixing my vscode settings, so I couldn't update it to single quotes